desktop/settings/linux.md

```diff
@@ -183,3 +171,21 @@ Use the **Extensions** tab to:
 - **Show Docker Extensions system containers**
 
 For more information about Docker extensions, see [Extensions](../extensions/index.md).
+
+## Feature control 
```

Comment:
Sorry, I probably missed this in the latest designs, I'd like to keep this 'Features in Development' for this first release since that was the framing we user tested. I know it's very different but when we user tested "Admin controls" naming people had a lot of negative associations with "controls" and when we add in the GA portion i'd like to user test before hand cc @gercekarmagan @lizzthabet can we update that on the feature as well?